src/components/file-component/FileComponent.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  fileWrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignSelf: 'stretch'
+  },
+  file: {
+    display: 'flex',
+    padding: '12px 16px',
+    alignItems: 'center',
+    gap: '16px',
+    alignSelf: 'stretch'
+  },
+  fileInfo: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    flex: '1 0 0'
+  },
+  fileDescription: {
+    display: 'flex',
+    '& p': {
+      color: 'primary.500',
+      typography: TypographyVariantEnum.Caption,
+      '&:first-of-type': {
```

Comment:
In the future, it can be another element as first, so better to create a separate style only for this element.
 Or you can create this circle by giving borderRadius: '50%', backgroundColor and some width with height for typography and pass this typography as the second element

Reply:
> In the future, it can be another element as first, so better to create a separate style and set it to only the first element. Or you can create this circle by giving borderRadius: '50%', backgroundColor and some width with height for typography and pass this typography as the second element

made separate style